libpod/network/cni/cni_exec.go

```diff
@@ -67,6 +68,10 @@ func (e *cniExec) ExecPlugin(ctx context.Context, pluginPath string, stdinData [
 	c.Stdout = stdout
 	c.Stderr = stderr
 
+	if !rootless.IsRootless() {
```

Comment:
Could you drop a comment here? The explanation in the commit message is great!

Reply:
done